libraries/helper.rb

```diff
@@ -47,8 +49,8 @@ def with_http_rescue(&block)
   end
 
   # exchanges a refresh token into an access token
-  def retrieve_access_token(server, refresh_token)
-    _success, _msg, access_token = Compliance::API.post_refresh_token(url, refresh_token, options['insecure'])
+  def retrieve_access_token(server_url, refresh_token, insecure = true)
```

Comment:
Can we set the default for insecure to false here? I think we should assume secure configurations in our code.